src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery1 = new Lottery();
```

Comment:
Why do you have`lottery1`? Do you have any another `Lottery` instance? Maybe simpler - e.g. `lottery`?